tests/test_pr_inversion.py

```
@@ -0,0 +1,886 @@
+        cls.data = make_synthetic_data('sphere', SPHERE_PARS, SPHERE_Q, seed=42)
+        cls.data.qmin = 0.01  # leave some excluded points
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore')
+            cls.result = pri.invert(cls.data, SPHERE_D_MAX, n_terms=10, alpha=1e5)
+            cls.scan = pri.explore_dmax(cls.data, SPHERE_D_MAX, n_terms=10, alpha=1e5, n_points=4)
+
+    def test_plot_pr_distribution(self):
+        fig = plot_pr_distribution(self.result, show=False)
+        names = [trace.name for trace in fig.data]
+        self.assertIn('P(r)', names)
+        self.assertIn('±1σ', names)
```

Ruff (0.16.1)

[warning] String contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)